repair_quotation_manual_sync/models/repair_order.py

```diff
@@ -0,0 +1,21 @@
+from odoo import api, fields, models
+
+
+class RepairOrder(models.Model):
+    _inherit = "repair.order"
+
+    sale_confirmed = fields.Boolean(
+        compute="_compute_sale_confirmed",
+        store=False,
+    )
```

Comment:
```suggestion
    is_sale_confirmed = fields.Boolean(
        compute="_compute_sale_confirmed"
    )
```
same as the others, no need to add the `store=False`. Also, WDYT about adding the prefix `is_` so it is easier to identify.